lib/modules/manager/npm/post-update/pnpm.spec.ts

```diff
@@ -11,6 +12,8 @@ jest.mock('../../../../util/exec/env');
 jest.mock('../../../../util/fs/proxies');
 jest.mock('./node-version');
 
+const readFixture = (x: string): string => Fixtures.get(x, '.');
+
```

Comment:
```suggestion
```
Why this, you don't need to pass baseDir by default, it's local `__fixtures__` directory by default and all fixtures should be in that directory. So this wrapper function should be removed.